bin/deploy-pr.sh

```diff
@@ -49,18 +50,18 @@ if [ -n "$CI_PULL_REQUESTS" ] && [ "$WEBCHANGES" -ne 0 ]; then
   aws s3api put-bucket-policy --bucket $BUCKET --policy "$BUCKET_POLICY"
   cd ../..
 
-  # Delete comment on Github, if there's one already
+  # Update the comment on Github, if there's one already.  This way we'll know the bot updated the thing.
   COMMENTS=$(curl -s -u "$GH_BOT_USER:$GH_BOT_PASSWORD" https://api.github.com/repos/18f/cms-hitech-apd/issues/$PRNUM/comments | jq -c -r '.[] | {id:.id,user:.user.login}' | grep "$GH_BOT_USER" || true)
   if [ "$COMMENTS" ]; then
     ID=$(echo "$COMMENTS" | jq -c -r .id)
-    curl -s -u "$GH_BOT_USER:$GH_BOT_PASSWORD" -X DELETE "https://api.github.com/repos/18f/cms-hitech-apd/issues/comments/$ID"
+    # Use $ before the body to use ANSI encoding, which preserves the newlines.
+    # Add the commit SHA so we know it deployed and when.
+    curl -s -u "$GH_BOT_USER:$GH_BOT_PASSWORD" -d $'{"body":"See this pull request in action: '"$DEPLOYED_URL"'\n\n'"$CIRCLE_SHA1"'"}' -H "Content-Type: application/json" -X PATCH "https://api.github.com/repos/18f/cms-hitech-apd/issues/comments/$ID"
```

Comment:
By using a `PATCH`, we can update the comment.  🎉  This sends basically the same message but with the git SHA appended.

Reply:
:fancy-cat: